src/decide.ts

```diff
@@ -74,6 +74,23 @@ export class Decide {
             this.instance['compression'] = compression
         }
 
+        // Check if recorder.js is already loaded
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        const surveysGenerator = window?.generateSurveys
+
+        if (response['surveys'] && !surveysGenerator) {
+            loadScript(this.instance.get_config('api_host') + `/static/surveys.js`, (err) => {
+                if (err) {
+                    return console.error(`Could not load surveys script`, err)
+                }
+
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                window.generateSurveys(this.instance)
```

Comment:
Can we scope this a little more perhaps? `window.extendPostHogWithSurveys` or something? 

Should be incredibly low chance of a clash but still. "generateSurveys" is somewhat too generic

Reply:
Makes sense, will do 👍